fly/integration/watch_test.go

```diff
@@ -111,6 +111,31 @@ var _ = Describe("Watching", func() {
 		})
 	})
 
+	Context("with just timestamp configuration", func() {
+		BeforeEach(func() {
+			atcServer.AppendHandlers(
+				ghttp.CombineHandlers(
+					ghttp.VerifyRequest("GET", "/api/v1/builds"),
+					ghttp.RespondWithJSONEncoded(200, []atc.Build{
+						{ID: 3, Name: "1", Status: "started", JobName: "some-job"},
+						{ID: 3, Name: "3", Status: "started"},
+						{ID: 2, Name: "2", Status: "started"},
+						{ID: 1, Name: "1", Status: "finished"},
+					}),
+				),
+				eventsHandler(),
+			)
+		})
+
+		It("watches the most recent one-off build with '--timestamp' configuration", func() {
+			watch("--timestamps")
```

Comment:
:thinking: These tests don't seem to verify anything timestamp-related, and it looks a little awkward to work timestamp tests in since `watch` just embeds its own assertions. If you're up for it, feel free to change how this is all done, but considering the meat of the functionality is already tested in the renderer, I'd be OK with merging this without the additional layer of tests.